tox.ini

```diff
@@ -1,26 +1,12 @@
 [tox]
 envlist = docs, py3, py27, py27-integration, flake8, flake8-py3, openstack
 
```

Comment:
The envlist from the `[tox]` section should be cleaned from the py27 targets.

Reply:
btw, flake description should be replaced with flake8-py3 I guess.

Reply:
fixed.

Reply:
nice, what about flake8?

Reply:
ahh, removed!